src/components/SystemHealthIcon/index.tsx

```diff
@@ -1,4 +1,9 @@
 import React from 'react';
+import {
+  IconCancel,
```

Comment:
I think these icons correspond to what's already in the application, however they do not match with the original Figma designs.  These icons were only used because we had to free options with Font-Awesome.  Now that we have Truss' icons, we can use the outlined versions of these icons.

`IconCheckCircleOutline,
  IconErrorOutline,
  IconHighlightOff`
  
Here is a link to the [Figma](https://www.figma.com/file/RhVQgXxE6tyj33TygymuhD/System-List-and-Profile-(Hi-fi)?node-id=1246%3A39874)